lms/static/js/inline_analytics.js

```diff
@@ -323,11 +357,15 @@ window.InlineAnalytics = (function() {
 
                     choiceCounter += 1;
                 }
-
+                // Construct the First Attempt row
+                firstCount = result[index]['first_count'];
+                firstPercent = Math.round(firstCount * 1000 / (totalFirstAttemptCount * 10));
+                firstCountRow += '<td class="checkbox_first_attempt">' + firstCount + '<br/>' + firstPercent + '%</td>'
```

Comment:
It's looping through each result from the api and building up the row by appending each time.